### PR TITLE
add hostname info

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Description

Just add a few collectors for we are able to get further information into the support bundle to check hostname

Screenshot 2023-02-17 at 13 49 09

### CODE REVIEW

File: host/cluster-down.yaml [hunk 1]
```diff
@@ -257,6 +257,18 @@ spec:
         collectorName: "crictl-logs-haproxy-previous"
         command: "sh"
         args: ["-c", "crictl logs -p $(crictl ps -a --name haproxy -l --quiet) 2>&1"]
+    # Gathering hostname info to help troubleshoot scenarios where the hostname mismatch
```
Comment (banjoh) [inline host/cluster-down.yaml:260]: Would it be cleaner and perhaps easier to read the collected information if it were in the same file?

This spec should achieve that

```yaml
    - run:
        collectorName: "hostnames"
        command: "sh"
        args:
          - -c
          - |
            echo "\$HOSTNAME = $HOSTNAME"
            echo "hostname = $(hostname)"
            echo "/proc/sys/kernel/hostname = $(cat /proc/sys/kernel/hostname)"
            echo "uname -n = $(uname -n)"
```

Expected output

```
$HOSTNAME = evans-mac.local
hostname = evans-mac.local
/proc/sys/kernel/hostname = 
uname -n = evans-mac.local
```

File: host/cluster-down.yaml [hunk 2]
```diff
@@ -257,6 +257,18 @@ spec:
...
+        collectorName: "hostname-envvar"
+        command: "echo"
+        args: ["$HOSTNAME"]
+    - run:
+        collectorName: "hostname-command"
+        command: "hostname"
+    - run:
+        collectorName: "proc-hostname"
+        command: "cat"
+        args: ["proc/sys/kernel/hostname"]
```
Comment (banjoh) [inline host/cluster-down.yaml:271]: Should be `/proc/sys/kernel/hostname`

**Review by banjoh — CHANGES_REQUESTED** [review]
Requesting some changes

File: host/cluster-down.yaml (hunk 1 above)
Comment (camilamacedo86) [reply host/cluster-down.yaml:260]: The envvar does not work in any way possible so we cannot add this one.

**Comment by danj-replicated** [comment]
I believe we already get the hostname from `uname` and store it in `host-collectors/system/hostos_info.json`?

**Comment by camilamacedo86** [comment]
HI @danj-replicated, 

> I believe we already get the hostname from uname and store it in host-collectors/system/hostos_info.json?

We do not check the values within. therefore, the goal is to have all data to see for example in a scenario where we face mismatch the values are not the same. 

echo "hostname = $(hostname)"
echo "/proc/sys/kernel/hostname = $(cat /proc/sys/kernel/hostname)"

**Review by danj-replicated — APPROVED** [review]

**Comment by banjoh** [comment]
Don't you still need to collect `$HOSTNAME`?

**Comment by banjoh** [comment]
> Don't you still need to collect $HOSTNAME?

This is a bash specific variable. We could do something like `/usr/bin/env bash -c 'echo "$HOSTNAME"'` but its not necessary for now.

**Review by banjoh — APPROVED** [review]

**Pull request merged**